src/stylesheet.css

```diff
@@ -1,6 +1,12 @@
 .player-button {
     padding: 0 4px;
 }
```

Comment:
Don't need the player-button class. It's not used by anything git rid of it and I'll merge this PR.